### PR TITLE
Add to workflow UX improvements

### PR DESCRIPTION
**Pull request by thecalcc**
SDBELGA-799
SDESK-7252

### CODE REVIEW

**Comment by tomaskikutis** [comment]
there should be a space between labels

image

File: client/components/Coverages/CoverageItem.tsx [hunk 1]
```diff
@@ -120,8 +123,13 @@ export class CoverageItemComponent extends React.Component<IProps, IState> {
             userAssigned: null,
             displayContentType: '',
             coverageDateText: '',
+            addedToWorkflow: false,
```
Comment (tomaskikutis) [inline client/components/Coverages/CoverageItem.tsx:126]: what about using `addedToWorkflow: coverage.workflow_status === COVERAGES.WORKFLOW_STATE.ACTIVE` so you don't need a block below?

File: client/utils/planning.ts [hunk 2]
```diff
@@ -1224,7 +1224,9 @@ function getCoverageIcon(
 }
 
 function getCoverageIconColor(coverage: IPlanningCoverageItem): string {
-    if (get(coverage, 'assigned_to.state') === ASSIGNMENTS.WORKFLOW_STATE.COMPLETED) {
+    if (coverage.workflow_status === COVERAGES.WORKFLOW_STATE.ACTIVE) {
```
[the remark this answers is not in the document]
Comment (thecalcc) [reply client/utils/planning.ts:1227]: No, fritz wants to have this one

[the remark this answers is not in the document]
Comment (thecalcc) [reply client/utils/planning.ts:1227]: No, fritz wants this one

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**